src/main/java/com/cloudant/client/org/lightcouch/Changes.java

```diff
@@ -215,7 +212,6 @@ private void setNextRow(Row nextRow) {
     }
 
     private void terminate() {
-        httpGet.abort();
```

Comment:
Have you seen any issues with this? Do we need to consider whether we can cancel a `HttpConnection` or something?